api/src/main/webapp/WEB-INF/queue.yaml

```diff
@@ -4,8 +4,8 @@ queue:
   target: api
 
   # rate parameters
-  bucket_size: 500
-  rate: 1/m
+  bucket_size: 1
+  rate: 10/m
```

Comment:
This is how quickly execution slots are refilled: 10 per minute.  Since the bucket size is now 1, this refill rate will be equal to the actual execution rate.